data_models/data_refinery_models/models/organism.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from xml.etree import ElementTree
+from django.db import models
+from data_refinery_models.models.base_models import TimeTrackedModel
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+NCBI_ROOT_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/"
+ESEARCH_URL = NCBI_ROOT_URL + "esearch.fcgi"
+EFETCH_URL = NCBI_ROOT_URL + "efetch.fcgi"
```

Comment:
I actually have a couple of questions here (even though I am reviewing the ``foreman/`` folder):
* So you decided to make a new Organism model instead of adding functionality to the one in ``django-organisms`` (https://bitbucket.org/greenelab/django-organisms)? I think that's totally fine, but any reason you decided to go this route?
* I do think that even if these URLs are relatively stable, it might be nice for the user to be able to override them in the ``settings.py`` file (but we can set these as defaults here in case the user does not specify them). I think this would be convenient in the future.